### PR TITLE
Talks and presentations page for Kueue

### PR DESCRIPTION
**Pull request by mimowo**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

To showcase Kueue based on publicly available materials and collect them in one place as a refence point.

#### Special notes for your reviewer:

This is a starting point, we can add more talks or presentations from wg-batch or other conferences.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by mimowo** [comment]
/hold
to avoid merging prematurely
/cc @tenzen-y @mwielgus @alculquicondor @dgrove-oss @astefanutti @ahg-g @kerthcet

**Comment by mimowo** [comment]
cc  @rochaporto

**Comment by rochaporto** [comment]
Awesome!

File: site/content/en/docs/talks_and_presentations/_index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+* KubeCon, Mar'24, Paris
+  [Advanced Resource Management for Running AI/ML Workloads with Kueue](https://youtu.be/6k_8Go3u8Qk)
+  Michał Woźniak, Google & Yuki Iwai, CyberAgent, Inc.
+
+* KubeCon, Mar'24, Paris
+  [Scale Your Batch / Big Data / AI Workloads Beyond the Kubernetes Scheduler](https://youtu.be/Ij5EAnuF-jk)
+  Antonin Stefanutti & Anish Asthana, Red Hat
+
+# 2023
```
Comment (kerthcet) [inline site/content/en/docs/talks_and_presentations/_index.md:30]: We also have a talk in KubeCon China with @Basasuya , https://www.youtube.com/watch?v=Q-sQLDMeJ8M

```
* KubeCon, Sep'27, Shanghai
  [Sailing Ray Workloads with KubeRay and Kueue in Kubernetes](https://youtu.be/Q-sQLDMeJ8M?si=QrLSN8I9Njmm8TVu)
   Jason Hu, Volcano Engine & Kante Yin, DaoCloud
```

**Comment by kerthcet** [comment]
> If that can be useful, the accompanying repository for our talk is available at https://github.com/astefanutti/kube-schedulers.

Ah, that's what I'm searching for, we're benchmarking kueue these days and compare with other batch systems. Thanks.

File: site/content/en/docs/talks_and_presentations/_index.md (hunk 1 above)
Comment (tenzen-y) [reply site/content/en/docs/talks_and_presentations/_index.md:30]: That is a great suggestion!

File: site/content/en/docs/talks_and_presentations/_index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    weight: 40
+description: >
+  Talks and presentations showcasing Kueue
+---
+
+# 2024
+
+* KubeCon, Nov'24, Salt Lake City
+  [Keynote: Multicluster Batch Jobs Dispatching with Kueue at CERN](https://youtu.be/xMmskWIlktA)
+  Ricardo Rocha, Lead Platforms Infrastructure, CERN & Marcin Wielgus, Staff Software Engineer, Google
```
Comment (tenzen-y) [inline site/content/en/docs/talks_and_presentations/_index.md:16]:
```suggestion
  Ricardo Rocha, CERN & Marcin Wielgus, Google
```

Are there any specific motivations the reason why we want to mention Job title only here?

File: site/content/en/docs/talks_and_presentations/_index.md (hunk 1 above)
Comment (mimowo) [reply site/content/en/docs/talks_and_presentations/_index.md:30]: Awesome! Updated

**Review by tenzen-y — COMMENTED** [review]
/lgtm

**Review by mwielgus — APPROVED** [review]
/lgtm

**Pull request merged**